test/binder-call/cmdline.l

```diff
@@ -0,0 +1,98 @@
+D              [0-9]
+l              "l"
+L              "L"
+u              "u"
+U              "U"
+h              "h"
+H              "H"
+COLON          ":"
+EIGHT          "8"
+
+INT8_SUFFIX    {COLON}{EIGHT}
+INT64_SUFFIX   [lL]
+UTF16_SUFFIX   [uU]
+HSTRING_SUFFIX [hH]
+
+%{
+#include <stdio.h>
+#include <glib.h>
+#include "cmdline.tab.h"
+
+#include "binder-call.h"
+#define YY_NO_INPUT
+#define YY_SKIP_YYWRAP
+#define YY_NO_UNPUT
+int cmdlinewrap(App *app);
+#undef yywrap
+#define yywrap() cmdlinewrap( (App*) args )
+#define YY_DECL int cmdlinelex( void *args )
+
+char *handle_str(char *text) {
+    char *str = g_strdup(text);
+    memmove(str, str+1, strlen(str)-1);
+    str[strlen(str)-3] = 0;
```

Comment:
What is that "-3"? and is it valid for both string16 and string8? Or is the use of handle_str() vs handle_str8() mixed up in rules below?

... I'm guessing 3 = leading quote + trailing quote + suffix - but the leading quote was already skipped and string8 does not have a suffix.

Reply:
There was actually something wrong logically. Thanks for noticing.